core/dbt/cli/params.py

```diff
@@ -133,13 +133,20 @@
     hidden=True,
 )
 
+models = click.option(
+    "-m",
+    "models",
+    envvar=None,
+    help="Specify the nodes to include.",
+    multiple=True,
+)
 
 output = click.option(
     "--output",
     envvar=None,
     help="TODO: No current help text",
     type=click.Choice(["json", "name", "path", "selector"], case_sensitive=False),
-    default="name",
+    default="selector",
```

Comment:
Matching default value of `"selector"` with what's in [main.py on the main branch](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/main.py#L844)